tools/slicec-cs/src/generators/enum_generator.rs

```diff
@@ -197,28 +197,16 @@ fn enumerators_as_nested_records(enum_def: &Enum) -> CodeBlock {
         let mut builder = ContainerBuilder::new("public partial record class", "Unknown");
 
         builder
-            .add_field("int Discriminant".to_owned())
-            .add_field("global::System.ReadOnlyMemory<byte> Fields".to_owned())
-            .add_base(enum_def.escape_identifier())
             .add_comment(
                 "summary",
                 format!(
                     "Represents an enumerator not defined in the local Slice definition of unchecked enum '{enum_name}'.",
                     enum_name = enum_def.identifier(),
                 ),
             )
-            .add_comment_with_attribute(
-                "param",
-                "name",
-                "Discriminant",
-                "The discriminant of this unknown enumerator.",
-            )
-            .add_comment_with_attribute(
-                "param",
-                "name",
-                "Fields",
-                "The encoded fields of this unknown enumerator.",
-            )
+            .add_field("Discriminant", "int", Some("The discriminant of this enumerator."))
```

Comment:
The previous doc comment had 'unknown' in it's description.
```suggestion
            .add_field("Discriminant", "int", Some("The discriminant of this unknown enumerator."))
```
Feel free to ignore if this was intentional, but just know we kept the 'unknown' in the doc comment underneath this one.